pandas/plotting/_core.py

```diff
@@ -526,30 +527,31 @@ class PlotAccessor(PandasObject):
         - 'area' : area plot
         - 'pie' : pie plot
         - 'scatter' : scatter plot
-        - 'hexbin' : hexbin plot
+        - 'hexbin' : hexbin plot.
+
     figsize : a tuple (width, height) in inches
     use_index : bool, default True
-        Use index as ticks for x axis
+        Use index as ticks for x axis.
     title : str or list
         Title to use for the plot. If a string is passed, print the string
         at the top of the figure. If a list is passed and `subplots` is
         True, print each item in the list above the corresponding subplot.
     grid : bool, default None (matlab style default)
-        Axis grid lines
+        Axis grid lines.
     legend : False/True/'reverse'
```

Comment:
```suggestion
    legend : bool or {'reverse'}
```

Reply:
Thanks for these comments @datapythonista. That link is very useful. I hadn't seen that pandas docstring guide before and it's extremely helpful for a new contributor. Do you think it might be a good idea to make it more visible on the doc website? Maybe add it in as a link on the right hand side tool bar? At the moment the link is in amongst the intro paragraph in the 'About the pandas documentation' section. I'd be happy to add this in if you agree.